docs/v3/how-to-guides/automations/creating-automations.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: Creating automations
+description: Learn how to define and manage automations.
+---
+
+## Create an automation
+
+On the **Automations** page, select the **+** icon to create a new automation. You'll be prompted to configure:
+
+- A [trigger](#triggers) condition that causes the automation to execute.
+- One or more [actions](#actions) carried out by the automation.
+- [Details](#details) about the automation, such as a name and description.
+
+import { COMBINED } from "/snippets/resource-management/combined.mdx"
+import { automations } from "/snippets/resource-management/vars.mdx"
+
+<COMBINED name="automations" hrefTF={automations.tf} hrefAPI={automations.api} hrefCLI={automations.cli} />
+
+
+### Create automations with the Python SDK
+
+You can create and access any automation with the Python SDK's `Automation` class and its methods.
+```python
+from prefect.automations import Automation
+from prefect.events.schemas.automations import EventTrigger
+from prefect.events.actions import CancelFlowRun
+
+# creating an automation
+automation = Automation(
+    name="woodchonk",
+    trigger=EventTrigger(
+        expect={"animal.walked"},
+        match={
+            "genus": "Marmota",
+            "species": "monax",
+        },
+        posture="Reactive",
+        threshold=3,
+    ),
+    actions=[CancelFlowRun()],
+).create()
+print(automation)
+# name='woodchonk' description='' enabled=True trigger=EventTrigger(type='event', match=ResourceSpecification(__root__={'genus': 'Marmota', 'species': 'monax'}), match_related=ResourceSpecification(__root__={}), after=set(), expect={'animal.walked'}, for_each=set(), posture=Posture.Reactive, threshold=3, within=datetime.timedelta(seconds=10)) actions=[CancelFlowRun(type='cancel-flow-run')] actions_on_trigger=[] actions_on_resolve=[] owner_resource=None id=UUID('d641c552-775c-4dc6-a31e-541cb11137a6')
+
+# reading the automation
+
+automation = Automation.read(id=automation.id)
+# or
+automation = Automation.read(name="woodchonk")
+
+print(automation)
+# name='woodchonk' description='' enabled=True trigger=EventTrigger(type='event', match=ResourceSpecification(__root__={'genus': 'Marmota', 'species': 'monax'}), match_related=ResourceSpecification(__root__={}), after=set(), expect={'animal.walked'}, for_each=set(), posture=Posture.Reactive, threshold=3, within=datetime.timedelta(seconds=10)) actions=[CancelFlowRun(type='cancel-flow-run')] actions_on_trigger=[] actions_on_resolve=[] owner_resource=None id=UUID('d641c552-775c-4dc6-a31e-541cb11137a6')
+```
+
+
+## Creating deployment triggers
```

Comment:
This section feels like it should potentially be its own how-to page.

Reply:
ah yeah that's a good point